File: ConstructAndDestructCPP/Source.cpp
//Copyright (c) 2021 RodKingroo

#include "BitString.h"
#include <iostream>

using namespace std;

int main() {

	BitString a("11");
	BitString b("1110");
	cout << "a=";
	a.output();
	cout << "b=";
	b.output();

	BitString c = a.conjaction(b);
	cout << "a&b=";
	c.output();

}

File: ClassesPython/Source.py
#Copyright (c) 2021 RodKingroo

from BitString import BitString

import sys

a = BitString()
b = BitString()

# print("a: ", end = ' ')
# a.output()
# print("b: ", end = ' ')
# b.output()
# print("a & b: ", end = ' ')
# c.output()

if len(sys.argv)<2:
    a.bitInput()
    b.bitInput()

else: 
    a.filebitInput(sys.argv[1])
    b.filebitInput(sys.argv[2])

c = a.conjaction(b)

print("Результат: ", end = ' ') 
c.output()

if len(sys.argv) == 4:
    c.fileOutput(sys.argv[3])